src/components/NavBar/NavBar.tsx

```diff
@@ -6,7 +6,7 @@ const NavBar: React.FC = () => {
   const texts = useTexts();
 
   return (
-    <AppBar position="static">
+    <AppBar position="fixed">
```

Comment:
Broke the homepage; Don't make such changes without verifying in other components. Besides, I don't think its in the wireframes that we got a fixed navbar 
![image](https://user-images.githubusercontent.com/14894196/100996302-90100180-3561-11eb-8b10-b651ce26e6fd.png)